### PR TITLE
test: World bank examples

### PR DESCRIPTION
**Pull request by kstrz**
### SUMMARY
Replace World Bank data example with data loaded by fixture for tests. The aim was to get rid of an external dependency. 

### TEST PLAN
All tests should be green.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by willbarrett — APPROVED** [review]
Looks good to me, would love for @dpgaspar to take a pass as well.

File: tests/dashboards/commands_tests.py [hunk 1]
```diff
@@ -78,79 +82,148 @@ def test_export_dashboard_command(self, mock_g1, mock_g2):
...
                 "DASHBOARD_VERSION_KEY": "v2",
             },
-            "metadata": {
-                "timed_refresh_immune_slices": [],
-                "expanded_slices": {},
-                "refresh_frequency": 0,
-                "default_filters": "{}",
-                "color_scheme": None,
-            },
+            "metadata": {"mock_key": "mock_value"},
```
Comment (dpgaspar) [inline tests/dashboards/commands_tests.py:226]: Why the change?

**Review by dpgaspar — COMMENTED** [review]
LGTM, but we still have some tests failing

File: tests/dashboards/commands_tests.py (hunk 1 above)
Comment (kstrz) [reply tests/dashboards/commands_tests.py:226]: I think the old value was the side effect of some other tests. Now I set `{"mock_key": "mock_value"}` explicitly in the fixture.
